seafobj/objstore_factory.py

```diff
@@ -103,6 +104,26 @@ def get_config_parser(self):
                 raise InvalidConfigError(str(e))
         return self.cfg
 
+    def get_seaf_crypto(self):
+        if not self.cfg.has_option('store_crypt', 'key_path'):
+            return None
+        key_path = self.cfg.get('store_crypt', 'key_path')
+        key_config = ConfigParser.ConfigParser()
+        try:
+            key_config.read(key_path)
+        except Exception:
+            return None
```

Comment:
If the option `key_path` is specified but we fail to the file, we should raise an exception here, instead of ignoring it.